SchleswigGottorf/GotD782.xml

```diff
@@ -0,0 +1,84 @@
+                <title>ʾArgānona wǝddāse</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="SD"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt><p>Cataloguing funded by the AHRC-DFG Project
+                  'Demarginalizing medieval Africa: Images, texts, and identity
+                  in early Solomonic Ethiopia (1270-1527)', grant ref.
+                  nos. AH/V002910/1 - 448410109</p></editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0.</licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0405Gottorf_Castle_"/>
+                        <idno>D 782</idno>
+                    </msIdentifier>
+                    <msContents>
+                       <summary/>
+                       <msItem xml:id="ms_i1">
+                        <title ref="LIT1146Argano"/>
+                         <textLang mainLang="gez"/>
+                      </msItem>
+                    </msContents>
+                   <physDesc>
+                     <objectDesc form="Codex">
+                        <supportDesc>
+                           <support>
+                              <material key="parchment"/>
+                           </support>
+                            <foliation>The manuscript is not foliated.</foliation>
+                         </supportDesc>
+                     </objectDesc>
+                      </physDesc>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Liturgy"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="SD" when="2022-04-06">Created entity with rudimentary
```

Comment:
i assume you are moving this ms from new? i do not see the deletion in the PR. you cannot create the same ms in a new directory without removing it from its old place

Reply:
Indeed. Removed.